content/datasets/sr_bh_2020.md

```diff
@@ -0,0 +1,110 @@
+---
+title: SR-BH 2020
+---
+
+- [Overview](#overview)
+- [Environment](#environment)
+- [Activity](#activity)
+- [Contained Data](#contained-data)
+- [Papers](#papers)
+- [Links](#links)
+- [Data Examples](#data-examples)
+
+| <!-- -->                 | <!-- -->                                   |
+|--------------------------|--------------------------------------------|
+| **Network Data Source**  | Custom network features                    |
+| **Network Data Labeled** | Yes                                        |
+| **Host Data Source**     | -                                          |
+| **Host Data Labeled**    | -                                          |
+|                          |                                            |
+| **Overall Setting**      | Single OS                                  |
+| **OS Types**             | Undisclosed                                |
+| **Number of Machines**   | 1                                          |
+| **Total Runtime**        | 12 days                                    |
+| **Year of Collection**   | 2020                                       |
+| **Attack Categories**    | Web-based attacks                          |
+| **User Emulation**       | Real users                                 |
+|                          |                                            |
+| **Packed Size**          | -                                          |
+| **Unpacked Size**        | 436 MB                                     |
+| **Download Link**        | [goto](https://doi.org/10.7910/DVN/OGOIXX) |
+
+***
+
+### Overview
+The SR-BH 2020 dataset (meaning of acronym unknown) consists of twelve days of traffic collected from a honeypot machine running a Wordpress webserver.
```

Comment:
I suppose it's the surnames of the first two authors

Reply:
Should I add this assumption here or just leave it out entirely?

Reply:
eh, I'll just add it, why not